### ISSUE TITLE
Pressing Escape when Adding Notes to a Task should NOT Discard the Notes

### ISSUE THREAD

**Issue by joshuajeschek** [nextcloud/tasks#2221]
In my opinion, pressing the escape button should function as simply exiting the notes field and keeping the content, instead of discarding the content. 
It has already happened multiple times now that I've typed up a big note, just for muscle memory (probably from vim) to kick in.
Pressing ESC has already deleted many precious words that I had to type again.
I'm sure many will agree with me on this :)

### Steps to reproduce
1. Create a new Task
2. Add Notes to it
3. Press Escape

### Expected behaviour
The notes should be saved

### Actual behaviour
The notes are discarded

**Comment by raimund-schluessler** [comment]
The opposite might also be true. Starting to edit the note and deleting/changing something by accident, pressing ESC gives you the option to abort the edit.

In my opinion, pressing ESC should abort whatever you did and not save it.

**Comment by JordanDIG** [comment]
> The opposite might also be true. Starting to edit the note and deleting/changing something by accident, pressing ESC gives you the option to abort the edit.
> 
> In my opinion, pressing ESC should abort whatever you did and not save it.

Yes for me this is what is missing in the application, also when you click on a task the use of the key could be to leave the task in progress as with the calendar.

**Comment by rokejulianlockhart** [comment]
@raimund-schluessler, in which case, an edit log is the solution to that issue.
